tests/integration/einstein/test_operations.py

```diff
@@ -222,3 +223,42 @@ def tearDown(self):
         self.db.project.delete_with_name(PROJECT)
         self.db.close()
         self.good_bmi.shutdown()
+
+
+class TestCopyImage(TestCase):
+    """
+    Creating a flatten copy of an image
+    """
+    @trace
+    def setUp(self):
+        self.db = Database()
+        self.db.project.insert(PROJECT, NETWORK)
+
+        self.good_bmi = BMI(CORRECT_HIL_USERNAME, CORRECT_HIL_PASSWORD,
+                            PROJECT)
+        self.good_bmi.import_ceph_image(EXIST_IMG_NAME)
+        self.img2 = 'img2'
```

Comment:
Can you make this a constant similar to PROJECT, EXIST_IMG_NAME, etc.

Reply:
not sure about it. I think it must be defined in CLI. Following is the list of variables available in CLI.  We should ask them to add a new variavle like IMG2 for us.

* picasso_url
* correct_hil_username
* correct_hil_password
* incorrect_hil_password
* node_name
* nic
* project
* network
* exist_img_name
* new_snap_name
* not_exist_snap_name
* not_exist_img_name